src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-credit-card/braintree-credit-card.ts

```diff
@@ -1,80 +1,107 @@
-var dropinInstance;
-
 import BraintreePaymentForm from '../braintree-payment-form/braintree-payment-form';
 import dropin from 'braintree-web-drop-in';
 
+interface BraintreeBillingAddress {
+    givenName: string;
+    surname: string;
+    phoneNumber: string;
+    streetAddress: string;
+    extendedAddress: string;
+    locality: string;
+    region: string;
+    postalCode: string;
+    countryCodeAlpha2: string;
+}
```

Comment:
Here are the same types for every value. So you can use `[key: string]: string;` instead.